python/route_guide/README.md

```diff
@@ -0,0 +1,281 @@
+
+This tutorial provides a basic Python programmer's introduction to working with gRPC. By walking through this example you'll learn how to:
+
+- Define a service in a .proto file.
+- Generate server and client code using the protocol buffer compiler.
+- Use the Python gRPC API to write a simple client and server for your service.
+
+It assumes that you have read the [Getting started](https://github.com/grpc/grpc-common) guide and are familiar with [protocol buffers] (https://developers.google.com/protocol-buffers/docs/overview).
+
+## Why use gRPC?
+
+This example is a simple route mapping application that lets clients get information about features on their route, create a summary of their route, and exchange route information such as traffic updates with the server and other clients.
+
+With gRPC you can define your service once in a .proto file and implement clients and servers in any of gRPC's supported languages, which in turn can be run in environments ranging from servers inside Google to your own tablet, with all the complexity of communication between different languages and environments is handled for you by gRPC. You also get all the advantages of working with protocol buffers, including efficient serialization, a simple IDL, and easy interface updating.
+
+## Example code and setup
+
+The example code for this tutorial is in [grpc/grpc-common/python/route_guide](https://github.com/grpc/grpc-common/tree/master/python/route_guide). To download the example, clone the `grpc-common` repository by running the following command:
+```shell
+$ git clone https://github.com/google/grpc-common.git
+```
+
+Then change your current directory to `grpc-common/python/route_guide`:
+```shell
+$ cd grpc-common/python/route_guide
+```
+
+You also should have the relevant tools installed to generate the server and client interface code - if you don't already, follow the setup instructions in [the Python quick start guide](https://github.com/grpc/grpc-common/tree/master/python).
+
+## Defining the service
+
+Your first step (as you'll know from [Getting started](https://github.com/grpc/grpc-common)) is to define the gRPC *service* and the method *request* and *response* types using [protocol buffers](https://developers.google.com/protocol-buffers/docs/overview). You can see the complete .proto file in [`grpc-common/protos/route_guide.proto`](https://github.com/grpc/grpc-common/blob/master/protos/route_guide.proto).
+
+To define a service, you specify a named `service` in your .proto file:
+
+```protobuf
+service RouteGuide {
+   // (Method definitions not shown)
+}
+```
+
+Then you define `rpc` methods inside your service definition, specifying their request and response types. gRPC lets you define four kinds of service method, all of which are used in the `RouteGuide` service:
+
+- A *bidirectionally-unary RPC* where the client sends a request to the server using the stub and waits for a response to come back, just like a normal function call.
```

Comment:
"Simple" struck me as a problematic term because it doesn't actually inform the reader about what it's describing or what the contrasting meaning of "complex" would be. It's only after seeing "request-streaming", "response-streaming", and "bidirectional streaming" that the reader learns that "simple" describes unary cardinality in both request and response directions, rather than something about invocation or timeouts or cascading.

But it's not worth maintaining a difference from the rest of the codebase. Changed to use "simple".